base_comment_template/models/res_partner.py

```diff
@@ -1,19 +1,14 @@
+# Copyright 2020 NextERP Romania SRL
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    property_comment_template_id = fields.Many2one(
+    base_comment_template_ids = fields.Many2many(
```

Comment:
You are losing here the `company_dependent` property, which is important. Migration scripts are needed for converting data from many2one to many2many (and with property) if we change to m2m, but I don't know the use case for requiring a m2m. Where that was changed?

Reply:
Commented in https://github.com/OCA/reporting-engine/pull/481#issuecomment-821799830, it's OK to change m2m + multi-company security, but migration scripts should be provided from those on 14.0.1.0.0 to go to this approach, and not fail when migrating from 13.0.1.0.0 to 14.0.1.0.0.

Reply:
OK but first, it's important to apply m2m in 13.0 in other PR, right?